mhkit/power/quality/gen_test_data.m

```diff
@@ -0,0 +1,102 @@
+function [i_m,u_m]=gen_test_data(Un,In,fg,fs,fm,fv,DeltaI_I,opt,T)
```

Comment:
This function was for testing the flicker code. I do not think it needs to be included in MHKiT

Reply:
1. It is written according to IECTS to test for performance of the generation of u0 and u_fic, user can use it to test their method settings (e.g., window settings for retrieving electrical angle of u_m using STFT, especially for conditions when ZCD does not work).
2. current testing of the code relies on the testing data generated by this code, I think it would be better to keep it documented for further reference.